pontoon/sync/changeset.py

```diff
@@ -347,7 +347,8 @@ def bulk_create_translations(self):
             target=t.string,
             locale_id=t.locale_id,
             entity_id=t.entity.pk,
-            translation_id=t.pk
+            translation_id=t.pk,
+            project=t.entity.resource.project_id,
```

Comment:
There's already `self.db_project`, let's use it instead. It should save us a bunch of DB hits.